src/hooks/screen/useCreateChat.ts

```diff
@@ -42,17 +42,24 @@ const useCreateChat = () => {
       channel_type: 'messaging',
       channel: channelWithMember,
       created_at: response?.channel,
-      targetName: selectedUser?.user?.username || selectedUser?.user?.name,
-      targetImage: selectedUser?.user?.profilePicture || selectedUser?.user?.image
+      targetName: selectedUser?.user?.username || selectedUser?.user?.name || selectedUser.username,
+      targetImage:
+        selectedUser?.user?.profilePicture ||
+        selectedUser?.user?.image ||
+        selectedUser.profilePicture
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for handling cases where `selectedUser.user` might not have the properties `username`, `name`, `profilePicture`, and `image`. However, it's important to note that this could potentially introduce inconsistencies if `selectedUser.username` or `selectedUser.profilePicture` differ from `selectedUser.user.username`, `selectedUser.user.name`, `selectedUser.user.profilePicture`, and `selectedUser.user.image`. If these properties are expected to always be consistent, it would be better to ensure they are synchronized at the source rather than handling it here.



<!-- This is an auto-generated comment by OSS CodeRabbit -->